redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisProviderConfig.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.statistic;
+
+import com.facebook.airlift.configuration.Config;
+
+public class RedisProviderConfig
+{
+    private String coordinator;
+    private String serviceName;
+    private String serverUri;
+    private String clusterPassword;
+    private long totalFetchTimeoutMs;
+
+    private long defaultTtlSeconds;
+    private boolean hboProviderEnabled;
+
+    private boolean clusterModeEnabled;
+    private long totalSetTimeoutMs;
+    private String credentialsPath;
+    private String redisPropertiesPath;
+
+    @Config("hbo.provider.cluster-mode-enabled")
```

Comment:
Please update the naming of all properties to include Redis HBO, e.g. 

`@Config("hbo.redis-provider.cluster-mode-enabled")`